Documentation/Registry.md

```diff
@@ -256,19 +253,15 @@ Package scopes are case-insensitive
 #### 3.6.2 Package name
 
 A package's *name* uniquely identifies a package in a scope.
-The maximum length of a package name is 128 characters.
+The maximum length of a package name is 100 characters.
 A valid package name matches the following regular expression pattern:
 
 ```regexp
-\A\p{XID_Start}\p{XID_Continue}{0,127}\z
+\A[a-zA-Z\d_.-]{1,100}\z
```

Comment:
Please see https://github.com/apple/swift-evolution/pull/1410#discussion_r673370637. 

I can see names beginning with `_` and `.` but not so much with `-`. Want to see what others think.

Reply:
`.` is used as a separator between scope and name, so we probably shouldn't allow it as the first character (to avoid, for example, `mona.........LinkedList`.